maxim/models/dataset.py

```
@@ -24,21 +32,51 @@ def dict_to_class(cls, data: Dict[str, Any]) -> "DatasetRow":
+    def from_json(cls, data: Dict[str, Any]) -> "Variable":
+        return cls(type_=data["type"], payload=data["payload"])
+
 
 class DatasetEntry:
+    """
+    This class represents a dataset entry.
+    """
+
     def __init__(
         self,
         input: Variable,
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)